x-pack/plugins/triggers_actions_ui/public/application/internal/shareable_components_sandbox/rule_event_log_list_sandbox.tsx

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { getRuleEventLogListLazy } from '../../../common/get_rule_event_log_list';
+
+export const RuleEventLogListSandbox = () => {
+  const props: any = {
+    rule: {
```

Comment:
@JiaweiWu, It seems the rule run history table needs only the rule id, it wouldn't be better to pass only, e.g. `ruleId` as a top-level prop instead of a rule object?

Reply:
Yea I definitely get what you're saying, we're trying to be more consistent with our props for all of our shared components, so making the change here would mean changing all of our other shared components (`rule_error_log` for example, which also only uses the rule ID, but takes the entire rule as a prop). 

So I feel like that is outside the scope of this PR, but I'm definitely open to hearing the entire teams thoughts on this so I t think we should create a separate issue to track this, if you feel very strongly about it.